f/FRRouting/build_info.json

```diff
@@ -0,0 +1,18 @@
+{
+    "maintainer": "Stuti.Wali@ibm.com",
+    "package_name": "FRRouting",
+    "github_url": "https://github.com/FRRouting/frr",
+    "version": "frr-9.0.1",
+    "default_branch": "master",
+    "package_dir": "f/FRRouting",
```

Comment:
@stutiibm reason behind creating folder with Org name and package name?
Also why we have different package names in build_info and build script?

Reply:
FRRouting directory is already available in "https://github.com/ppc64le/build-scripts",  if I put "package_dir": "f/frr" it was giving error as "/home/jenkins/build-scripts/f/frr does not exist." in test_jenkins_job. So kept "FRRouting" as package name in build_info.json

Also, when I clone github repository of package it get cloned as "frr" directory. So kept "frr" as package name in build-script.

Reply:
To resolve above issue, I am renaming directory in https://github.com/ppc64le/build-scripts from FRRouting to frr.